src/hw_isolation_record/manager.cpp

```diff
@@ -830,8 +830,7 @@ void Manager::handleHostIsolatedHardwares()
             {
                 entryIt->second->resolveEntry(false);
             }
-            else if (std::distance(validEntryRecords.begin(),
-                                   validEntryRecords.end()) == 1)
+            if(std::ranges::distance(validEntryRecords) == 1)
```

Comment:
why a change in this line..?

Reply:
Just as a better practice updated it. But I will revert it for now.

Reply:
done